src/controller/project/options.go

```diff
@@ -19,22 +19,31 @@ type Option func(*Options)
 
 // Options options used by `Get` method of `Controller`
 type Options struct {
-	CVEAllowlist bool // get project with cve allowlist
-	Metadata     bool // get project with metadata
-	WithOwner    bool
+	WithCVEAllowlist       bool // get project with cve allowlist
+	WithEffectCVEAllowlist bool // get project with effect cve allowlist
```

Comment:
What's the difference between `CVEAllowlist` and `EffectCVEAllowlist` ?

Reply:
`CVEAllowlist` is the allow list associated with the project in the table `cve_allowlist`, and `EffectCVEAllowlist` is the allow list in system-level when the project's `reuse_sys_cve_allowlist` metadata is true otherwise it's the allow list associated with the project in the table `cve_allowlist`.